lib/index.js

```diff
@@ -200,13 +199,25 @@ class PWMetrics {
 
   findMedianRun(results) {
     const ttiValues = results.map(r => r.timings.find(timing => timing.id === metrics.ids.TTI).timing);
-    const medianTTI = median(ttiValues);
+    const medianTTI = this.getMedianRun(ttiValues);
     // in the case of duplicate runs having the exact same TTI, we naively pick the first
     const medianRun = results.find(result => result.timings.find(timing => {
```

Comment:
a little odd we now have `findMedianRun()` `getMedianRun()` and `medianRun`.

afaict, your new `getMedianRun` is just getting a median value of the TTIs passed in, but not returning a run.

can we clean this up?

i'm open to solutions, but one would be we nuke this last `results.find(result => result... ` bit and and pass the `results` into your new getMedianRun. In there, we operate on each `result` and compare its TTI result value to the other. and in the end we return the result object.

but if you want do it differently, that WFM too. :)